kustomize/skipper/flagger/patch.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: flagger
+spec:
+  template:
+    spec:
+      containers:
+        - name: flagger
+          args:
+            - -log-level=info
+            - -mesh-provider=kubernetes
+            - -metrics-server=http://flagger-prometheus:9090
+            - -slack-user=flagger
```

Comment:
we should probably remove these lines when not used

Reply:
thanks - removed